src/current/_includes/v24.3/ldr/multiple-tables.md

```diff
@@ -0,0 +1 @@
+There are some tradeoffs between enabling one table per LDR job versus multiple tables in one LDR job. Multiple tables in one LDR job can be easier to operate. For example, if you pause and resume the single job, LDR will stop and resume for all the tables. However, the most granular level observability will be at the job level. One table in one LDR job will allow for table-level observability.
```

Comment:
I feel like this note combines operational consideration with observability. Should we keep it scoped to observability considerations given that this appears on the observability page only?

Reply:
So, I have now added this "include" file to the tutorial page as well. It just ensures that if we change the advice in any way, I can update this one file and it will update anywhere. I felt like it might be ok on the monitoring page, as it is also talking about job operation (which we kind of touch on in Monitoring LDR).